ansible/roles/master/templates/kubectl.kubeconfig.j2

```diff
@@ -5,7 +5,7 @@ preferences: {}
 clusters:
 - cluster:
     certificate-authority-data: {{ kube_ca_cert|b64encode }}
-    server: https://{{ groups['masters'][0] }}:{{ kube_master_api_port }}
+    server: https://{{ kube_apiserver_endpoint }}:{{ kube_master_api_port }}
```

Comment:
I think this can change to the following so we do not need to introduce the kube_apiserver_endpoint var:

```
server: https://{% if groups['masters']|length > 1 %}{{ kube_apiserver_vip }}{% else %}{{ groups['masters'][0] }}{% endif %}:{{ kube_master_api_port }}
```